wikimedia/wikipedia-ios
# Native Editor - add comment support

**Phabricator:** 
https://phabricator.wikimedia.org/T348072

### Notes
This wraps up T348072 with comment syntax highlighting and button actions.

### Test Steps
1. Open editor on Staging
2. Confirm comments are syntax highlighted as gray
3. Tap within a comment. Confirm comment button on formatting input view is in a selected state.
4. Tap button. Confirm comment markup is removed.
5.  Tap button again, confirm comment markup is added.
